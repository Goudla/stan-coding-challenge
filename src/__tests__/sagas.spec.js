import rootSaga from '../sagas';

describe('sagas', () => {
  it('should export root saga', () => {
    const generator = rootSaga();
    const value = generator.next().value;
    expect(value).toMatchSnapshot();
  });
});
