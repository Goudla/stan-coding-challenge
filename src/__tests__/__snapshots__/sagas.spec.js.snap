// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sagas should export root saga 1`] = `
Object {
  "@@redux-saga/IO": true,
  "combinator": true,
  "payload": Array [
    Object {
      "@@redux-saga/IO": true,
      "combinator": false,
      "payload": Object {
        "args": Array [],
        "context": null,
        "fn": [Function],
      },
      "type": "FORK",
    },
  ],
  "type": "ALL",
}
`;
